src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/EntityDtoValidator.java

```diff
@@ -131,7 +131,7 @@ private boolean validatePublicRegisterJurisdiction(String publicRegisterName, St
         var compoundQualifiedFieldName = String.format("%s and %s", qualifiedFieldNamePublicRegisterName, qualifiedFieldNamePublicRegisterJurisdiction);
 
         return StringValidators.isNotBlank(publicRegisterJurisdiction, qualifiedFieldNamePublicRegisterJurisdiction, errors, loggingContext)
-                && StringValidators.isLessThanOrEqualToMaxLength(String.format(CONCATENATED_STRING_FORMAT, publicRegisterName, publicRegisterJurisdiction), 160, compoundQualifiedFieldName, errors, loggingContext)
+                && StringValidators.isLessThanOrEqualToMaxLength(String.format(CONCATENATED_STRING_FORMAT, publicRegisterName, publicRegisterJurisdiction), 159, compoundQualifiedFieldName, errors, loggingContext)
```

Comment:
Just had a look. The concatenated string it supposed to be 160 or less, i.e. with the comma, which means the user input should be 159 to account for it, so I'm not sure this should change as we are testing the string as whole. If we want to test user input only then we'd need to remove the concatenated string format, append the two string and then set it to 159. Any error messages both in the api and the web should specify "user input" when stating 159. @markpit and @sarahocarroll what do you think?

Reply:
i'll pull the branch and have a look, i was hoping it would just be a case of updating the error message text.

Reply:
Yes, what Matt said should work. instead of using the CONCATENATED_STRING_FORMAT to join the 2 strings together, you could just create a new constant - CONCATENATED_STRING_FORMAT_NO_COMMA = %s%s
That will then check the length of the 2 strings together and use the 159 value like you have done.